docs/gcp.md

```diff
@@ -0,0 +1,166 @@
+# Deploying to Google Cloud Platform (GCP)
+
+This guide describes how to build and run your Swift Server on serverless
+architecture with [Google Cloud Build](https://cloud.google.com/build) and
+[Google Cloud Run](https://cloud.google.com/run). We'll use
+[Artifact Registry](https://cloud.google.com/artifact-registry/docs/docker/quickstart)
+to store the Docker images.
+
+## Google Cloud Platform Setup
+
+You can read about
+[Getting Started with GCP](https://cloud.google.com/gcp/getting-started/) in
+more detail. In order to run Swift Server applications, we need to:
+
+- enable [Billing](https://console.cloud.google.com/billing) (requires a credit
+  card)
+- enable the
+  [Cloud Build API](https://console.cloud.google.com/apis/api/cloudbuild.googleapis.com/overview)
+- enable the
+  [Cloud Run Admin API](https://console.cloud.google.com/apis/api/run.googleapis.com/overview)
+- enable the
+  [Artifact Registry API](https://console.cloud.google.com/apis/api/artifactregistry.googleapis.com/overview)
+- [create a Repository in the Artifact Registry](https://console.cloud.google.com/artifacts/create-repo)
+  (Format: Docker, Region: your choice)
+
+## Project Requirements
+
+Please verify that your server listens on `0.0.0.0`, not `127.0.0.1` and it's
+recommended to use the environment variable `$PORT` instead of a hard-coded
+value. For the workflow to pass, two files are essential, both need to be in the
+project root:
+
+1. Dockerfile
+2. cloudbuild.yml
+
+You should test your Dockerfile with `docker build . -t test` and
+`docker run -p 8080:8080 test` and make sure it builds and runs locally.
+`cloudbuild.yml` contains a set of steps to build the server image directly in
+the cloud and deploy a new Cloud Run instance after the successful build.
```

Comment:
Move the cloud build description to where the file is
```suggestion
`docker run -p 8080:8080 test` and make sure it builds and runs locally.
```

Reply:
Ping

Reply:
Missed that, thanks!